nginx-auth0/nginx.conf

```diff
@@ -155,12 +160,24 @@ http {
          if res.email then
            ngx.log(ngx.DEBUG, "bearer_jwt_verify response: mail ", res.email)
            ngx.req.set_header("X-Akvo-Email", res.email)
+           ngx.var.akvoemail = res.email
          else
            ngx.say(cjson.encode({error="Invalid access_token"}))
            ngx.exit(ngx.HTTP_FORBIDDEN)
 	 end
       }
 
+      proxy_cache http_204;
+      proxy_cache_revalidate off;
+      proxy_cache_background_update on;
+      proxy_cache_lock on;
+      proxy_cache_methods GET;
+      proxy_cache_key $proxy_host$request_uri$akvoemail;
+      proxy_cache_valid 204 60m;
```

Comment:
I was thinking more in 30 to 60 secs. "Real time"

Reply:
For _"real time"_ we have a way to _punch a whole_ and bypass the cache. My intention with the 1h cache is to avoid reaching the database and other services in a transparent way. The client is not aware that we're serving cached content.